frame/child-bounties/src/benchmarking.rs

```diff
@@ -0,0 +1,302 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020-2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Child-bounties pallet benchmarking.
+
+#![cfg(feature = "runtime-benchmarks")]
+
+use super::*;
+
+use frame_benchmarking::{account, benchmarks, whitelisted_caller};
+use frame_system::RawOrigin;
+
+use crate::Pallet as ChildBounties;
+use pallet_bounties::Pallet as Bounties;
+use pallet_treasury::Pallet as Treasury;
+
+const SEED: u32 = 0;
+const MAX_BYTES: u32 = 16384;
```

Comment:
Should use something like `MaximumReasonLength` instead, also see https://github.com/paritytech/substrate/pull/10368